### DISCUSSION TITLE
Let's see how far we can go...

### DISCUSSION THREAD

**Discussion by Leaf5323** [DavidS95/Smokeless_UMAF#53] [Show and tell]
I guess this is just one successful example of how much more performance we can squeeze out from our "**outdated**" laptops 😂

![ZenTimings_Screenshot](https://github.com/DavidS95/Smokeless_UMAF/assets/74171766/00a6e274-4465-46a8-a23a-fd9e99ce3a44)

**Comment by Espionage724** [comment]
I accidentally added an extra 133MHz on my RAM speed :p (can't seem to overclock or change timings but surprisingly some Q ref setting 100 or 133 applied)

Not sure if I've seen performance improvements, but I get faster boots by disabling legacy USB deep in chipset/SA settings! (1s faster POST)

XTU/overclocking menu lets me apply offset undervolts at BIOS and every OS easy :D

**Comment by ahmet20400-design** [comment]
How did you do it? I bricked my Asus Tuf Gaming FX705DT by increasing ram speed to 3200 MHz. Now its black screen.
